extensions/binding/org.eclipse.smarthome.binding.sonos/README.md

```diff
@@ -2,6 +2,12 @@
 
 This binding integrates the [Sonos Multi-Room Audio system](http://www.sonos.com).
 
+**Attention:** 
+You might run into trouble if your Eclipse Smarthome instance is in another subnet than your Sonos device. 
```

Comment:
Better talk of "your gateway" or something neutral - remember that those docs are e.g. taken over to the openHAB documentation website.

Reply:
I was thinking about that, too. But „gateway“ is not a good idea either as that is widely used in another context in networking.

Reply:
Then simply use "instance".